src/app/components/pages/EventDetails.tsx

```diff
@@ -52,16 +51,23 @@ import { IsaacContent } from "../content/IsaacContent";
 import { EditContentButton } from "../elements/EditContentButton";
 import { MapContainer, Marker, Popup, TileLayer } from "react-leaflet";
 import * as L from "leaflet";
-
-function formatDate(date: Date | number) {
-  return dayjs(date).format("YYYYMMDD[T]HHmmss");
-}
+import { Location } from "../../../IsaacApiTypes";
 
 interface EventDetailsProps {
   match: { params: { eventId: string } };
   location: { pathname: string };
 }
 
+export const formatAddress = (location: Location | undefined) => {
```

Comment:
Didn't we already have a function that was very similar to this one somewhere else?

Reply:
Yes we had a similar one, though outputting a slightly differently formatted result, in another Event component. For simplicity I've combined them and put them into a shared location.